boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -396,6 +397,7 @@ WinLdrLoadModule(PCSTR ModuleName,
     ArcClose(FileId);
     if (Status != ESUCCESS)
     {
+        WARN("Error while reading %s, Status: %u\n", ModuleName, Status);
```

Comment:
s/reading/closing/

Reply:
@SergeGautherie read the code carefully, the `Status` is returned here from `ArcRead()` at line 396. If you don't care about reading the code, your review is useless.

Reply:
You are right, I missed this case is different: my bad, sorry.
No need to lecture me on that, is it?